### ISSUE TITLE
time

### ISSUE THREAD

**Issue by 18022443868** [CoinCheung/BiSeNet#122]
start1 = cv2.getTickCount()
out = net(im)[0].argmax(dim=1).squeeze().detach().cpu().numpy()
end = cv2.getTickCount()
tt1 = (end - start1) / (cv2.getTickFrequency()) * 1000
print('te = ', tt1, 'ms')

Local test time： 742ms  RTX2080
Where did I configure it wrong?

**Comment by CoinCheung** [comment]
Is this the first time of inference ? Maybe you should consider running inference for 1000 times and compute the average.

**Comment by CoinCheung** [comment]
I am closing this since no further information provided. Maybe you have solved your problem now.
